tests/publiccloud/check_registercloudguest.pm

```diff
@@ -100,13 +99,7 @@ sub run {
         last if ($instance->ssh_script_output(cmd => 'zypper -x lr | grep "<repo" | wc -l', timeout => 300) == 0);
         sleep 15;
```

Comment:
To **count** the _Enabled_ repositories, we could also use the -e export option in the command:
`zypper lr -E -e - | grep -c 'name='`
to could all, insted:
`zypper lr -e - | grep -c 'name='`
:blush:

Reply:
this a bit unrelated to main purpose of this PR . there is always plenty of things which can be improved but let's do them step by step . Feel free to open PR with suggestion :)